cypress/integration/remote-storage/configurations/crud.spec.js

```diff
@@ -0,0 +1,127 @@
+import { bigtestGlobals } from '@bigtest/globals';
+import { Link } from '@bigtest/interactor';
+import {
+  Button,
+  KeyValue,
+  Modal,
+  MultiColumnList,
+  MultiColumnListCell,
+  Pane,
+  Select,
+  TextField
+} from '../../../../interactors';
+
+bigtestGlobals.defaultInteractorTimeout = 10000;
+
+describe('making CRUD', () => {
+  before(() => {
+    cy.visit('/');
+
+    cy.login('diku_admin', 'admin');
+
+    // We rely here on Settings routing,
+    // but navigate inside our own module-under-test using UI
+    cy.visit('/settings/remote-storage');
+    cy.do(Link('Configurations').click());
+  });
+
+  it('should provide happy path', function () {
+    const NAME = 'Test name 1';
+    const NAME_EDITED = 'Test name 2 - edited';
+
+    const PROVIDER = 'Dematic EMS';
+    const PROVIDER_CODE = 'DEMATIC_EMS';
+
+    // Create
+    cy.do(Button('+ New').click())
+      .expect([
+        Pane('Create configuration').exists(),
+        TextField('Remote storage name\n*').is({ focused: true, value: '' }),
+        Select('Provider name\n*').has({ value: '' }),
+        Button('Cancel').exists(),
+        Button('Save & close').exists(),
```

Comment:
This is redundant since you'll be using this button later. In most cases, you do not need `exists()` assertions.